### PR TITLE
Minor edits to the gravity benchmark section.

### PR DESCRIPTION
**Pull request by bangerth**
Update to #4164, with just minor stuff.

I replaced $R_s$ by $r$ because that is what the beginning of the paragraph uses for the distance.

@cedrict FYI.

/rebuild

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -9567,14 +9567,14 @@ \subsubsection{Thin shell gravity benchmark}
 -164391.6151 &
 -164392.2246 &
 -164390.9963 \\
-&+1 & $3^3$ &  
+& 1 & $3^3$ &
```
Comment (tjhei) [inline doc/manual/manual.tex:9570]: Is this n_q? Would it be helpful to add a header to this table?

**Review by tjhei — APPROVED** [review]
Looks good, except the one comment. 👍

File: doc/manual/manual.tex (hunk 1 above)
Comment (cedrict) [reply doc/manual/manual.tex:9570]: oups, for some reason there I left out 2 '+' signs which serve no purpose. And also the values in the column below are 0.

**Comment by bangerth** [comment]
@tjhei If you're willing to merge this PR, then I will let @cedrict add the table header separately.

**Comment by tjhei** [comment]
Some tests are failing. Is master broken?

**Comment by tjhei** [comment]
@gassmoeller some Steinberger tests are failing on master. Do you know what is the reason?

**Comment by bobmyhill** [comment]
Master was broken briefly but it should work again now. #4203 fixed the tests.

**Comment by bangerth** [comment]
I rebased. Let's see if that fixes the tests.

**Comment by gassmoeller** [comment]
Looks good, but you picked up a conflict to current master, please rebase again.

**Comment by bangerth** [comment]
Rebased.

**Pull request merged**